.github/workflows/coverage.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           fetch-depth: 1
 
       - name: create instrumentation coverage
-        uses: ReactiveCircus/android-emulator-runner@v2.8.0
+        uses: ReactiveCircus/android-emulator-runner@v2
```

Comment:
lets be specific about the version, much like how we avoid using `+` notation in gradle dependencies

Reply:
I don't believe its a nice idea. As if we keep it just `@v2`, then the runner would automatically get updated as they(reactive-circus) roll out the new versions(which sometimes may take care of deprecations) of `2`.

Reply:
We don't want automatic updates, when we upgrade we want it to be intentional, in this scenario if I reran a build it would not be the same build as when it first ran in the case where an update happens after the initial build. Builds should be repeatable and we have no guarantee future versions work with our configuration.

Reply:
Gotcha!